flooddrake/flux.py

```diff
@@ -17,14 +19,17 @@ def Interior_Flux(N, V, wr, wl):
 
     d = V.mesh().geometric_dimension()
 
-    gravity = parameters["flooddrake"]["gravity"]
+    gravity = Function(V.sub(0)).assign(parameters["flooddrake"]["gravity"])
```

Comment:
Ah yes, so you do cancel gravity in dry cells flux (hence why we need a Function rather than a Constant), see line 31 and 73.